src/_includes/blocks/footer.njk

```diff
@@ -1,11 +1,81 @@
- <footer class="border-t text-xs pt-4 mt-4">
+ <div class="text-xs mt-4 mx-4 mb-0">
+ The Globe Church is all about Jesus, made up of all sorts of people, involved in the greatest mission... 
+ </div>
+ 
+ <footer class="border-t text-xs pt-4 mt-3 mx-4">
```

Comment:
Can you add an `id="footer"` here too please? If Javascript doesn't load when someone clicks the Menu button we're going to drop them down here. The id attribute can be used for that…

`<a href="#footer">Menu</a>` -> `<footer id="menu" class="…">`

Reply:
Example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_an_element_on_the_same_page